cmd/promtool/main.go

```diff
@@ -124,7 +125,12 @@ func main() {
 		os.Exit(CheckMetrics())
 
 	case queryInstantCmd.FullCommand():
-		os.Exit(QueryInstant(*queryServer, *queryExpr, p))
+		{
+			if !strings.Contains(*queryServer, "http") {
```

Comment:
I am not aware of the placement of this check or any other ways, but if we are going with string matching, then I think only the prefix should be checked for `http://` and `https://` and not the entire URL for the existence of `http`.

Reply:
Hi, thanks for your comment, I change it :)